recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
If you have any ideas on how to better approach this problem I'd love to hear your advice. This is the recommended way according to the pysdl2 docs, but it does offer some other options: https://pysdl2.readthedocs.io/en/rel_0_9_5/integration.html#importing-pysdl2. If you tell me which one has preference I can see what I can do. 

What exactly do you mean by relocatable? That this package runs in a conda environment, wherever this is located?  That is the case this way, right? The sdl2 library is determined relative to the pysdl2 module's location, so it should work for any virtual environment.